doc/modules/cassandra/pages/developing/cql/SASI.adoc

```diff
@@ -405,15 +405,15 @@ or more page-sized blocks. The
 https://github.com/apache/cassandra/blob/trunk/src/java/org/apache/cassandra/index/sasi/disk/OnDiskIndex.java[`OnDiskIndex`]
 is structured as a tree of arrays, where each level describes the terms
 in the level below, the final level being the terms themselves. The
-`PointerLevel`s and their `PointerBlock`s contain terms and pointers to
+``PointerLevel``s and their ``PointerBlock``s contain terms and pointers to
```

Comment:
This is leaving the `` `s `` out of the highlight.
<img width="608" alt="Captura de pantalla 2024-12-27 a la(s) 8 22 10 a m" src="https://github.com/user-attachments/assets/9fac25c9-0005-4687-b840-c0e9efd69c17" />

Reply:
![SASI_2](https://github.com/user-attachments/assets/c432997e-1af8-40be-8412-5c78e03918bd)

fix renders properly after antora build